examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -37,8 +37,10 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        df = filter_K_core(df, min_rating=10, num_iter=1)
</pre><p>Probably you need more iterations here (you could try with different values of <code>num_iter</code> to see when you get at least 10 ratings for each user and item). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>